.github/workflows/release.yml

```diff
@@ -219,37 +233,45 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+
       - name: Install ARM target
         run: rustup update && rustup target add aarch64-apple-darwin
+
       - name: Import Certificate
-        uses: apple-actions/import-codesign-certs@v1
+        uses: apple-actions/import-codesign-certs@b3ad3190cbe076413153f9db48e0307b2ca694b3 # use action with node16, can be dropped after new action version
```

Comment:
@panekj This seems to be failing the release pipeline, can you please take a look?

Reply:
https://github.com/lapce/lapce/actions/runs/3428929033